LayoutTest.xcodeproj/project.pbxproj

```diff
@@ -677,6 +677,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				ENABLE_BITCODE = NO;
```

Comment:
With bitcode enabled, linking against `XCTest.framework` would fail because it does not contain bitcode.`